### PR TITLE
[API] Export layernorm and rmsnorm api.

### PR DESCRIPTION
**Pull request by changqi1**

### CODE REVIEW

File: include/layers.h
Comment (pujiang2018) [inline include/layers.h:1]: The file name "layers.h" is more like an abstraction of Layers. How about normalization.h?

**Review by pujiang2018 — COMMENTED** [review]
Just one small comment for the file name.

File: include/layers.h
Comment (changqi1) [reply include/layers.h:1]: I want to add all layers including normalization, MLP, attention and so on in this include/layers file to export to users.

Comment (pujiang2018) [reply include/layers.h:1]: Here are 2 options:
1) use a single file layers.h
2) use multiple files like layers_norm.h, layers_mlp.h, ...
My intuition is that the second option is better.

Comment (changqi1) [reply include/layers.h:1]: Sure.

**Comment by changqi1** [comment]
@pujiang2018 Modified file name.

**Pull request merged**